app/pulp/app/serializers/base.py

```diff
@@ -0,0 +1,173 @@
+from pulp.app.apps import get_pulp_plugin_configs
+from rest_framework import serializers
+
+
+def view_name_for_model(model_obj, view_action):
```

Comment:
This function is no longer broken, but has been broken up into two actions, one to just get the viewset for a model, and the other to get the view name from that viewset.